src/main/kotlin/io/snyk/plugin/SnykPostStartupActivity.kt

```diff
@@ -78,11 +78,6 @@ class SnykPostStartupActivity : StartupActivity.DumbAware {
         LOG.info("Loading variants for all amplitude experiments")
         val experimentUser = ExperimentUser(publicUserId)
         service<AmplitudeExperimentService>().fetch(experimentUser)
-
-        if (service<AmplitudeExperimentService>().isShowScanningReminderEnabled()) {
-            SnykBalloonNotifications.showScanningReminder(project)
```

Comment:
Shell we remove `SnykBalloonNotifications.showScanningReminder` and correspondent code as well? Or we planning to reenable it later?
cc @pavel-snyk

Reply:
I'll just remove it and merge - we can always get it back from git.